projects/draco/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Google Inc.
```

Comment:
"git pull" and git push with --force so that these committed changes are not visible.